storm-client/src/jvm/org/apache/storm/daemon/StormCommon.java

```diff
@@ -85,11 +84,6 @@ public static StormCommon setInstance(StormCommon common) {
         return oldInstance;
     }
 
-    @Deprecated
```

Comment:
The method isn't really very useful. It's just as easy to just call getTopoId on the StormClusterState yourself, rather than using this method. I think that's a good enough reason to remove it.

Reply:
Yeah I don't have strong opinion on this, so OK to remove it. I haven't use it directly.